server/game/cards/01-Core/PowerBehindTheThrone.js

```diff
@@ -10,32 +10,17 @@ class PowerBehindTheThrone extends PlotCard {
         });
         this.action({
             title: 'Discard a stand token',
-            method: 'discardToken'
-        });
-    }
-
-    discardToken(player) {
-        if(!this.hasToken('stand')) {
-            return false;
-        }
-
-        this.game.promptForSelect(player, {
-            activePromptTitle: 'Select a character',
-            source: this,
-            cardCondition: card => card.location === 'play area' && card.controller === player && card.kneeled,
-            onSelect: (p, card) => this.onCardSelected(p, card)
+            cost: ability.costs.discardTokenFromSelf('stand'),
+            target: {
+                cardCondition: card => card.location === 'play area' && card.controller === this.controller && card.kneeled
```

Comment:
it looks like Power Behind the Throne can be use also to stand a character controlled by your opponent, so the controller check should be removed